datamodel/src/main/kotlin/com/hartwig/actin/datamodel/clinical/IHCTest.kt

```diff
@@ -14,4 +13,6 @@ data class PriorIHCTest(
     val scoreValue: Double? = null,
     val scoreValueUnit: String? = null,
     val impliesPotentialIndeterminateStatus: Boolean = false
-)
+) {
+    val test: String = IHC_TEST_TYPE
```

Comment:
I think you can just get rid of this entirely and set the test = "IHC" in the IHCTestInterpreter